include/z64actor.h

```diff
@@ -243,7 +243,11 @@ typedef struct Player {
     /* 0x03AC */ Actor*     heldActor;
     /* 0x03B0 */ char       unk_3B0[0x018];
     /* 0x03C8 */ Vec3f      unk_3C8;
-    /* 0x03D4 */ char       unk_3D4[0x060];
+    /* 0x03D4 */ char       unk_3D4[0x058];
+    /* 0x042C */ u8         unk_42C;
+    /* 0x042D */ s8         unk_42D;
+    /* 0x042E */ u16        unk_42E;
```

Comment:
These are actually signed (from my player decomp)
```suggestion
    /* 0x042C */ s8         unk_42C;
    /* 0x042D */ s8         unk_42D;
    /* 0x042E */ s16        unk_42E;
```